### ISSUE TITLE
FR: release binaries

### ISSUE THREAD

**Issue by pataquets** [pythops/tuix#2]

**Comment by pythops** [comment]
I will release once I get something close to stable which is not the case at this point

**Comment by pataquets** [comment]
Fair enough. Would you be open to merge a Github Actions workflow which builds an executable on each git tag (even 0.x)?
If tagging does not suit, it can be also triggered on each commit, on crontab-style schedule or even manually at the press of a button.
No guarantees on ETA here, but at least it'll make your intentions visible to everyone else who might be willing.

**Comment by pythops** [comment]
sure, send the PR :)
